cluster/images/conformance/run_e2e.sh

```diff
@@ -59,7 +59,7 @@ ginkgo_args+=(
 )
 
 set -x
-/usr/local/bin/ginkgo "${ginkgo_args[@]}" /usr/local/bin/e2e.test -- --disable-log-dump --repo-root=/kubernetes --provider="${E2E_PROVIDER}" --report-dir="${RESULTS_DIR}" --kubeconfig="${KUBECONFIG}" | tee "${RESULTS_DIR}"/e2e.log &
```

Comment:
This script will need to define a default value; otherwise it will err because the variable is unset. You can also swap it out for `${E2E_VERBOSITY:4}` to provide the default inline.

Reply:
Thanks for your comment, @johnSchnake 
We can swap it as you said, but this PR just follows the other valuable ways.
We will be able to do that for cleanup later with other ones also with another PR.